pkg/kubelet/kubelet.go

```diff
@@ -2238,32 +2239,88 @@ func (kl *Kubelet) rejectPod(pod *api.Pod, reason, message string) {
 		Message: "Pod " + message})
 }
 
+func (kl *Kubelet) getNodeResources() api.ResourceList {
```

Comment:
nit: We are trying to break `kubelet` object into sub-modules. Can these changes be places behind a new interface, something like `NodeBuilder` for example?